classes/abstracts/ActionScheduler_Store.php

```diff
@@ -5,12 +5,13 @@
  * @codeCoverageIgnore
  */
 abstract class ActionScheduler_Store extends ActionScheduler_Store_Deprecated {
-	const STATUS_COMPLETE = 'complete';
-	const STATUS_PENDING  = 'pending';
-	const STATUS_RUNNING  = 'in-progress';
-	const STATUS_FAILED   = 'failed';
-	const STATUS_CANCELED = 'canceled';
-	const DEFAULT_CLASS   = 'ActionScheduler_wpPostStore';
+	const STATUS_COMPLETE       = 'complete';
+	const STATUS_PENDING        = 'pending';
+	const STATUS_RUNNING        = 'in-progress';
+	const STATUS_FAILED         = 'failed';
+	const STATUS_CANCELED       = 'canceled';
+	const DEFAULT_CLASS         = 'ActionScheduler_wpPostStore';
+	const IGNORE_GROUP_FLAG    = '--';
```

Comment:
It's unlikely of course that there are many existing group names with this prefix, but not impossible. Since there is no particular cost (to us) in terms of making this lengthier/more unique, perhaps we could consider:

```suggestion
	const IGNORE_GROUP_FLAG     = '--*IGNORE*--';
```

Or some similar variation. It's a small and easy change on our side, but reduces the potential for collisions with existing group names considerably (since, if I am reading correctly, method `group_has_ignore_prefix` won't guard against existing group names).